Add docs for Discover Homepage (#5618)
getsentry/sentry-docs: src/docs/product/discover-queries/index.mdx

### Delete Queries

On the **Discover** page, each saved query card has an ellipsis that will open a context menu. From here, you can delete the query. This action is irreversible. You can also delete the query within the [query results](#query-results) view by clicking the "Delete Saved Query" option in the context menu.

### Add to Dashboard

Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="react">custom dashboards</SandboxLink> as widgets. You can find the "Add to Dashboard" button by opening the context menu in the **Discover** page or within the context menu of the [query results](#query-results) view. When you click "Add to Dashboard", a form opens where you can name your widget, select your target dashboard, and make other changes before saving your new widget.

### Set as Default

<Include name="early-adopter-note.mdx" />

> If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query. You can customize your homepage by configuring how your columns and charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.

Reviewer: One last nit: we're not just configuring how the chart appears, we're possibly changing the query, right? Made the following suggestion.

```suggestion
If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query. 

You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.
```

Author: Yep! Everything you can set in a discover query, you can now save it for the homepage (except for the title because each case is a new query). I think this new copy makes sense and I've updated it

> If you don't want to use this configuration as your default homepage anymore, click "Remove Default"; the homepage will revert to the "All Events" query.

Reviewer: Changing this back as it flows better.

```suggestion
If you don't want to use this configuration as your default homepage anymore, click "Remove Default", and the homepage will revert to the "All Events" query.
```

### Add to Dashboard

Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="react">custom dashboards</SandboxLink> as widgets. You can find the "Add to Dashboard" button by opening the context menu in the **Discover** page or within the context menu of the [query results](#query-results) view. When you click "Add to Dashboard", a form opens where you can name your widget, select your target dashboard, and make other changes before saving your new widget.

### Set As Default

<Note>

The **Discover** homepage is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

> Queries can be used as your **Discover** homepage. You can find the "Set As Default" button by opening the context menu in the **Discover** page or within the [query results](#query-results). You can customize your homepage by configuring how your columns and charts appear, and then clicking "Set As Default". Navigating to **Discover** through the left sidebar will now open a new query builder with this configuration pre-loaded.

Reviewer: Since these two features (seeing the query builder as the homepage + being able to set a homepage) are part of the same feature, I'm rolling this all into one bit of content here. When this goes into GA, then it'll make sense to split it back up the way you had it before, with some tweaks.

```suggestion
If your organization is part of the Early Adopter program, when you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query. You can customize your homepage by configuring how your columns and charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.

If you don't want to use this configuration as your default homepage anymore, click "Remove Default"; the homepage will revert to the "All Events" query.

To navigate to the query management page, where you can find pre-built and saved queries, click the "Saved Queries" button.

```

> 
> If you don't want to use this configuration as your default homepage anymore, click "Remove Default" on the homepage and the homepage will revert to the "All Events" query.
> 

Reviewer: ```suggestion
```

Queries can also be added to <SandboxLink scenario="dashboards" projectSlug="react">custom dashboards</SandboxLink> as widgets. You can find the "Add to Dashboard" button by opening the context menu in the **Discover** page or within the [query results](#query-results) view. When you click "Add to Dashboard", a form opens where you can name your widget, select your target dashboard, and make other changes before saving your new widget.

### Use as Discover Home

<Note>

The Discover homepage is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

> Queries can be used as your Discover homepage. You can find the "Use as Discover Home" button by opening the context menu in the **Discover** page or within the [query results](#query-results). If you have a particular configuration you'd like to see on the homepage, configure your columns and charts, then click "Use as Discover Home". Navigating to Discover through the sidebar will now open a new query builder with these configurations pre-loaded.

Reviewer: ```suggestion
Queries can be used as your **Discover** homepage. You can find the "Use as Discover Home" button by opening the context menu in the **Discover** page or within the [query results](#query-results). You can customize your homepage by configuring how your columns and charts appear, and then clicking "Use as Discover Home". Navigating to **Discover** through the left sidebar will now open a new query builder with this configuration pre-loaded.
```

> If you do not wish to use this configuration as your homepage anymore, click "Reset Discover Home" on the homepage and the homepage will revert to the default All Events query.

Reviewer: ```suggestion
If you don't want to use this configuration anymore, click "Reset Discover Home" and the homepage will revert to the default "All Events" query.
```
